src/renderer/widgets/PasswordInputField.cpp

```diff
@@ -171,13 +172,17 @@ bool CPasswordInputField::draw(const SRenderData& data) {
 
     bool forceReload = false;
 
+    if (passwordLength == 0 && g_pHyprlock->getPasswordFailedAttempts() > failedAttempts)
```

Comment:
I found culprit, it was initially working wrong since I allowed updating faillasset on empty imput. The color function was broken and incorrectly spamming `animated = true` causing redraw. Now it works correct but nothing else triggers `return true` in this function since dots = passlen on empty input. Color function is being called but only once and in checking state, thus not triggering `animated` because colors are equal.

i guess this is harmless

Reply:
hwat

Reply:
nvm, all ok